src/react-components/button-link.tsx

```diff
@@ -24,6 +28,17 @@ export const ButtonLink = ({
   position,
   ...props
 }: ButtonLinkProps): JSX.Element => {
+  let handleClick
+
+  if (Array.isArray(href)) {
```

Comment:
why use an Array here? Couldn't we just create an `href` type with keys and values?

Reply:
or type guard it?

Reply:
Well, I didn't want to break BC, so  kept href as a string. I assume we could use an object instead of an array, sure.